File: Test_Coverage/processed_dataset/tests/test_HumanEval_13.py
from functions.HumanEval_13 import greatest_common_divisor

def test_HumanEval_13():
  assert greatest_common_divisor(3, 7) == 1
  assert greatest_common_divisor(10, 15) == 5
  assert greatest_common_divisor(49, 14) == 7
  assert greatest_common_divisor(144, 60) == 12

File: Test_Coverage_Java/src/test/java/test_HumanEval_13.java
import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class HumanEval_13Test {

    @Test
    public void testGreatestCommonDivisor() {
        assertEquals(1, functions.HumanEval_13.greatest_common_divisor(3, 7));
        assertEquals(5, functions.HumanEval_13.greatest_common_divisor(10, 15));
        assertEquals(7, functions.HumanEval_13.greatest_common_divisor(49, 14));
        assertEquals(12, functions.HumanEval_13.greatest_common_divisor(144, 60));
    }
}